consts/randomChaoItemList.go

```diff
@@ -36,7 +36,7 @@ var RandomChaoWheelCharacterPrizes = map[string]float64{
 	//enums.CTStrSticks:          0.3, // Revival Event
 	//enums.CTStrTikal:           1.5, // Event (Sonic Adventure)
 	enums.CTStrVector: 0.3,
-	enums.CTStrWerehog:         1.5, // Revival Event
+	// enums.CTStrWerehog:         1.5, // Revival Event
```

Comment:
Werehog should be kept on as a lower odds character. Remember that we aren't explicitly removing mainline characters from the roulette anymore.

Reply:
Sure. we can keep the Werehog in the Premium Roulette and normalize his odds